example/example/tests/test_grapple.py

```diff
@@ -483,17 +492,33 @@ def test_explicit_order(self):
             query, variables={"searchQuery": "Gamma", "order": "-title"}
         )
         page_data = executed["data"].get("pages")
+        print(page_data)
 
         self.assertEquals(len(page_data), 6)
         self.assertEquals(page_data[0]["title"], "Gamma Gamma")
+        self.assertEquals(page_data[3]["title"], "Gamma")
         self.assertEquals(page_data[1]["title"], "Gamma Beta")
         self.assertEquals(page_data[2]["title"], "Gamma Alpha")
-        self.assertEquals(page_data[3]["title"], "Gamma")
         self.assertEquals(page_data[4]["title"], "Beta Gamma")
         self.assertEquals(page_data[5]["title"], "Alpha Gamma")
 
 
 class PageUrlPathTest(BaseGrappleTest):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.child = BlogPageFactory(slug="child", parent=cls.home)
+        cls.parent = BlogPageFactory(slug="parent", parent=cls.home)
+        cls.parent_child = BlogPageFactory(slug="child", parent=cls.parent)
+
+        cls.another_home = HomePage.objects.create(
+            title="Another home", slug="another-home", path="00010002", depth=2
+        )
+        cls.another_site = wagtail_factories.SiteFactory(
+            site_name="Another site", root_page=cls.another_home
+        )
+        cls.another_child = BlogPageFactory(slug="child", parent=cls.another_home)
```

Comment:
changing this to 

```suggestion

    def setUp(self):
        super().setUp()

        self.another_site = wagtail_factories.SiteFactory(
            site_name="Another site", root_page=self.another_home
        )
        self.another_child = BlogPageFactory(slug="child", parent=self.another_home)
```

for some funny reason `self.another_site` is `self.child` in `test_with_multisite`, so that blows up

Reply:
alternatively, doing

```python
    def test_with_multisite(self):
        self.another_site.refresh_from_db()
```

seems to fix that

Reply:
I just moved it to it's own test method and it seems to work. I'm assuming there is some memo-ization happening somewhere.